src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -483,11 +499,14 @@ public DetailedPackageMetadata PackageMetadata
                         if (alternatePackage != null)
                         {
                             newAlternatePackageText = GetPackageDeprecationAlternatePackageText(alternatePackage);
+                            UriBuilder builder = new UriBuilder("nugetpm", alternatePackage.PackageId);
```

Comment:
Is the nugetpm part relevant in any way?

Reply:
I just use an URI because I am using a hyperlink to trigger PM UI search. Since this starts looking like deep linking, I just created an issue: https://github.com/NuGet/Home/issues/11032

I can change to another approach to pass packageId to command.

Reply:
Removed.